test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Indexers/FunctionIndexerTests.cs

```diff
@@ -214,11 +214,11 @@ public Task<ITriggerBinding> TryCreateAsync(TriggerBindingProviderContext contex
 
         [AttributeUsage(AttributeTargets.Parameter)]
         [Binding]
-        public class ExtensionTrigger : Attribute
+        public class ExtensionTriggerAttribute : Attribute
         {
             private string _path;
 
-            public ExtensionTrigger(string path)
+            public ExtensionTriggerAttribute(string path)
```

Comment:
Had to make these changes because my tests were failing. This is due to https://github.com/Azure/azure-webjobs-sdk/blob/5df91e0e6a23b08db0ff37ecdf803c7a145bda1b/src/Microsoft.Azure.WebJobs.Host/Extensions/JobHostMetadataProvider.cs#L128